examples/iot-dashboard/CMakeLists.txt

```diff
@@ -4,18 +4,20 @@
 cmake_minimum_required(VERSION 3.14)
 project(sixtyfps_cpp_iot_dashboard LANGUAGES CXX)
 
+if (NOT "cxx_std_20" IN_LIST CMAKE_CXX_COMPILE_FEATURES)
+    message(STATUS "The C++ iot dashboard demo requires C++ >= 20. The compiler you're using does not support that, so the build of this example is skipped.")
+    return()
```

Comment:
Either this test does not work, or the Ubuntu 20.04 compiler claim to support c++20  while it doesn't support format